index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+    <title>About Me</title>
+  </head>
+  <body>
+    <div class="mobile-container">
+      <nav class="topnav">
+        <a href="#home"><img src="./images/logo.png" alt="Home"></a>
+        <div id="myLinks">
+          <a href="#contact">Contact</a>
+          <a href="#about">About</a>
+        </div>
+        <a href="javascript:void(0);" class="icon" onclick="toggleNav()">
+			<img src="./images/hamburger.png" alt="Menu icon">
+        </a>
+      </nav>
+
+      <div class="content">
+        <h1>I am Siddhartha Glad to see you!</h1>
+        <p>I am a software developer!. I can help you build a product, feature or website.
+			 Look through some of my work and experience! 
+			 If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+        <p></p>
+		<p><a href="#">LET'S CONNECT</a></p>
+      </div>
+    </div>
+	<footer>
+		<div class="footer-container">
+		  <div class="social-media">
+			<a href="#">
+				<img src="./images/facebook.png" alt="Facebook">
+			  </a>
+			  <a href="#">
+				<img src="./images/linkedin.png" alt="LinkedIn">
+			  </a>
+			  <a href="#">
+				<img src="/images/twitter.png" alt="Twitter">
+			  </a>
+			  <a href="#">
+				<img src="./images/instagram.png" alt="Instagram">
+			  </a>
+		  </div>
```

Comment:
- [ ] kindly consider  wrapping up all the social media icons in a `ul` to make your code semantically correct and after that consider adding each icon in a `li`as they are lists it will also make your code semantically correct too

Reply:
@divinecharlotte I have wrapped them up inside ul and ui please have a look:

<div class="image-container ul">
        <ul>
          <ui>
            <a href="#">
              <img src="./images/facebook.png" alt="Facebook">
            </a>
          </ui>
        </ul>
        <ul>
          <ui>
            <a href="#">
              <img src="./images/linkedin.png" alt="LinkedIn">
            </a>
          </ui>
        </ul>
			  <ul>
          <ui>
            <a href="#">
              <img src="/images/twitter.png" alt="Twitter">
            </a>
          </ui>
        </ul>
			  <ul>
          <ui>
            <a href="#">
              <img src="./images/instagram.png" alt="Instagram">
            </a>
          </ui>
        </ul>
		  </div>